examples/llm/src/models/mosaic_gpt.py

```diff
@@ -179,18 +208,24 @@ def forward(
 
         attn_bias = self._attn_bias(device=x.device, dtype=x.dtype)
 
+        # initialize the past key values cache if it should be used
+        if use_cache and past_key_values is None:
+            past_key_values = [[] for _ in range(self.config.n_layers)
```

Comment:
```suggestion
            past_key_values = [() for _ in range(self.config.n_layers)
```


[here](https://github.com/mosaicml/examples/pull/243/files#diff-cb5960418e22962455775ab1498957a896b97bdfbc69eab8444ee0779d224939R80) the type'd return type shows tuple
It doesn't matter if you keep the suggestion, just stay consistent.